routes.php

```diff
@@ -64,4 +64,4 @@
 $router->add('/historical-class-search-results', 'attendance/historical_class_lookup_results.php', 'Historical Class Search Results');
 $router->add('/attendance-form-confirmed', 'attendance/attendance_form_confirmed.php', 'Attendance Confirmed');
 $router->add('/edit-participant', 'attendance/edit_participant.php', 'Edit Participant');
```

Comment:
This will cause a conflict when the participant management pull request is accepted. Both of the routes will be the same

Reply:
@vmjoseph 

I see where the conflict is, but this route has been there since attendance was merged into the branch. I think edit-participant may be too vague of a name, so I will change it to `attendance-edit-participant`. However, I think that the fact that we had this conflict means the page name `edit-participant` may not be detailed enough. I would suggest adding more detail to the page names. Perhaps `ps-edit-participant`.

Reply:
Understood. I'll make the addition to my end